xapian-applications/omega/docs/omegascript.rst

```diff
@@ -222,6 +222,11 @@ $fmt
 $freq{term}
 	frequency of a term
 
+$hash{HASH,TEXT}
+    unique ID for TEXT string using the hashing algorithm specified by HASH.
+    Currently, this command only supports MD5 hashing algorithm which can be
+    used by specifying HASH as "md5" (without quotes).
```

Comment:
Since `HASH` must be `md5` at the moment, I suspect it'd be better to put the hash argument second, and make it optional but defaulting to md5.

It's also perhaps worth noting which characters will be emitted by `$hash{}` (ie the hex characters), since that tells people whether they'll need escaping in various contexts. Also pointing out the length in characters of an md5 hash may be useful.

Finally, I'd put md5 in a "fixed-space literal" (surrounded by double backticks) rather than enquoting it then saying "(without quotes)".